manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: simonmichael.hledger
+PackageVersion: 1.29.2
+PackageLocale: en-US
+Publisher: simonmichael
+PublisherUrl: https://github.com/simonmichael/hledger
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: Simon Michael
+PackageName: hledger
+PackageUrl: https://github.com/simonmichael/hledger
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/simonmichael/hledger/blob/master/LICENSE
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Robust, fast, intuitive plain text accounting tool with CLI, TUI and web interfaces. 
+Description: hledger is lightweight, cross platform, multi-currency, double-entry accounting software. It lets you track money, investments, cryptocurrencies, invoices, time, inventory and more, in a safe, future-proof plain text data format with full version control and privacy.
+Moniker: hledger
+# Tags:
```

```suggestion
Tags:
- finance
- haskell
- accounting
- tui
- web-application
- ledger
- bounty
- plain-text
- command-line-tool
- console-application
- plaintext
- plaintext-accounting
- bounties
- haskell-application
- plain-text-accounting
```
Taking the opportunity to suggest some more useful metadata